framework/src/modules/random/module.ts

```diff
@@ -41,10 +42,13 @@ export class RandomModule extends BaseModule {
 		if (errors.length) {
 			throw new LiskValidationError(errors);
 		}
-		this._maxLengthRevealMainchain =
-			(moduleConfig as ModuleConfig).maxLengthRevealsMainchain ?? MAX_LENGTH_REVEALS_MAINCHAIN;
+		this._moduleConfig = moduleConfig as ModuleConfig;
+		this._maxLengthReveal =
+			(moduleConfig as ModuleConfig).maxLengthRevealsMainchain ?? DEFAULT_MAX_LENGTH_REVEALS;
```

Comment:
I think if you use `this._moduleConfig` here instead of `(moduleConfig as ModuleConfig)` then you don't repeat `(moduleConfig as ModuleConfig)` two times in init.

Reply:
By doing that also you don't need the console.log below

Reply:
i guess you can keep one of them (not necessary to duplicate the data)